component/federation.jsonnet

```diff
@@ -66,6 +69,17 @@ local scrape_config = kube.Secret('additional-scrape-configs') {
             ],
             target_label: 'namespace',
           },
+          // Put back original namespace label otherwise. This should work
+          // because replace doesn't overwrite existing labels.
+          {
+            action: 'replace',
+            regex: '(.*)',
+            replacement: '$1',
```

Comment:
`regex` and `replacement` can be ommited here I think

Reply:
according to the docs the field `regex`  is required for action `replace`: https://prometheus.io/docs/prometheus/latest/configuration/configuration/#relabel_config

Reply:
I'm asking because 20 lines up it's also ommited

Reply:
good point